src/ui/server/controllers/insurance/business/broker/index.test.ts

```diff
@@ -195,6 +198,23 @@ describe('controllers/insurance/business/broker', () => {
           expect(res.redirect).toHaveBeenCalledWith(expected);
         });
       });
+
+      describe('when an extra field is inserted onto the page', () => {
+        it('should call mapAndSave.broker once without the extra field', async () => {
```

Comment:
I don't think we need this as `constructPayload` will test the removal of the field, for each controller we can just update the existing call so e.g

```js
it('should call mapAndSave.broker once with data from constructPayload function', async () => {
```

Makes it simpler, what do you think?

Reply:
done